core/app/models/spree/product.rb

```diff
@@ -97,6 +97,9 @@ class Product < Spree::Base
 
     alias :options :product_option_types
 
+    self.whitelisted_ransackable_associations = %w[stores variants_including_master master variants]
```

Comment:
We may need to add `sku` to this list due to the following call to ransack in dotandbo_spree:
`Spree::Product.ransack("variants_including_master_sku_cont" => keywords).result`